APIScrape/scrape.php

```diff
@@ -20,10 +20,11 @@ function createDatabase(\PDO $db) {
                 CREATE TABLE `breed` (
                 `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
                 `breed` varchar(255) NOT NULL DEFAULT '',
+                `favourite_cat` int(11) unsigned DEFAULT NULL,
```

Comment:
We know it's a cat - this is the ID of the cat pic that has been marked as the favourite, so it should be `favourite_id`